src/components/chloropleth/__tests__/Chloropleth.spec.tsx

```
@@ -0,0 +1,270 @@
+    _path: string,
+    _index: number
+  ) => {
+    return <path key={`hover-${feature.properties.gemcode}`} />;
+  };
+
+  it('Should call the correct callbacks for the given feature collections', () => {
+    // Arrange
+    let featureCallbackWasCalled = false;
+    let overlayCallbackWasCalled = false;
+    let hoverCallbackWasCalled = false;
+
+    const featureCallback = (
+      feature: Feature<MultiPolygon, MunicipalityProperties>,
+      _path: string,
+      _index: number
+    ) => {
+      featureCallbackWasCalled = true;
+      expect(featureCollection.features.indexOf(feature)).toBeGreaterThan(-1);
+      return <path key={`feature-${feature.properties.gemcode}`} />;
+    };
+
+    const overlayCallback = (
+      feature: Feature<MultiPolygon>,
+      _path: string,
+      index: number
+    ) => {
+      overlayCallbackWasCalled = true;
+      expect(overlays.features.indexOf(feature)).toBeGreaterThan(-1);
+      return <path key={`overlay-${index}`} />;
+    };
+
+    const hoverCallback = (
+      feature: Feature<MultiPolygon, MunicipalityProperties>,
+      _path: string,
+      _index: number
+    ) => {
+      hoverCallbackWasCalled = true;
+      expect(featureCollection.features.indexOf(feature)).toBeGreaterThan(-1);
+      return <path key={`hover-${feature.properties.gemcode}`} />;
+    };
+
+    const props: TProps<MunicipalityProperties> = {
+      featureCollection,
+      overlays,
+      hovers,
+      boundingBox: boundingbox,
+      dimensions,
+      featureCallback,
+      overlayCallback,
+      hoverCallback,
+      onPathClick: defaultOnPathClick,
+      getTooltipContent: defaultGetTooltipContent,
+    };
+
+    // Act
+    const renderResult = render(<Chloropleth {...props} />);
+
+    let groups = null;
+    try {
+      groups = renderResult.getAllByText(
+        (_content: string, element: HTMLElement) => {
+          return element.nodeName === 'g';
+        }
+      );
+    } catch (e) {}
```

Don't you at least want do something like print the error? 

In my experience, failing silently is only useful if you act on it in the catch clause.